src/ui/components/search/filtersearchcomponent.js

```diff
@@ -223,14 +235,14 @@ export default class FilterSearchComponent extends Component {
    * optionally redirecting based on config. Uses window.setTimeout to allow
    * other filters to finish rendering before searching.
    */
-  search () {
+  search (resetPagination = true) {
```

Comment:
Did we need to add this flag for some bug unrelated to storage? What is a case we'd search without resetting pagination?

Reply:
on page reload or backnav, we don't want to resetPagination